aws/src/utils/safe_object_from_dict.py

```diff
@@ -16,12 +16,12 @@ def safe_object_from_dict(object_type: type[T], data: dict) -> T:
 
     for key in list(data.keys()):
         if key not in object_type.__annotations__.keys():
-            print(f"WARNING: {key} not in class {object_type.__name__}")
+            # print(f"WARNING: {key} not in class {object_type.__name__}")
```

Comment:
We should uncomment this incase other scripts find it useful - otherwise remove it in another PR :)

Reply:
Fixed in 5216e150eb7639e544d08afcb46958dcf840d444